### PR TITLE
fixes issue with losing styling

### PR DESCRIPTION
**Pull request by thetif**
fixes issue #2961 

### This pull request changes...

- the way the rich text is sanitized on save
- should be able to include styling (colors, spacing) on text

### This is how to verify this change...

- open an APD
- add text with formatting
- go back to the State Dashboard
- open the APD again
- the formatting should still be there

### This pull request is ready to merge when...

- [X] Develop has updated automated tests (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author
- [x] QA has manually tested and created bugs for any issues found (beyond minor fixes)
- [X] Changelog is updated as appropriate

### This feature is done when...

- [ ] QA has verified the accessibility and functionality related to the change
- [x] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: api/util/sanitize.test.js [hunk 1]
```diff
@@ -22,6 +22,17 @@ tap.test('sanitize input', async sanitizeTests => {
     test.equals(sanitize(dirty), dirty, 'allows link attributes');
   });
 
+  sanitizeTests.test('it should allow style for spans', async test => {
+    const dirty =
+      '<span style="color:#3598db;text-decoration:line-through">styling</span>';
+    test.equals(sanitize(dirty), dirty, 'alows style attributes');
```
Comment (tbolt) [inline api/util/sanitize.test.js:28]: small typo on allows here

**Review by tbolt — APPROVED** [review]

**Review by eStar386 — APPROVED** [review]
Passes: Functional and Accessibility testing.

**Review by beparticular — APPROVED** [review]
Everything seems to be saving fine now including bullets. Let me know if I need to look at anything else specific besides multiple types of formatting and bullets, if there's anything else specifically concerning

**Review by NAretakis — APPROVED** [review]
Good to go!

**Pull request merged**